src/base/CMakeLists.txt

```diff
@@ -168,3 +168,7 @@ if (APPLE)
     find_library(AppKit_LIBRARY AppKit)
     target_link_libraries(qbt_base PRIVATE ${Carbon_LIBRARY} ${IOKit_LIBRARY} ${AppKit_LIBRARY})
 endif (APPLE)
+
+if (WIN32)
+    target_link_libraries(qbt_base PRIVATE advapi32 iphlpapi ole32 powrprof shell32 user32 wsock32 ws2_32)
```

Comment:
PowrProf is already linked in https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/powermanagement/CMakeLists.txt

Reply:
Why do you ignore this comment?

Reply:
It's still actual...

Reply:
@jagannatharjun 
please remove `PowrProf` here for now..

Reply:
you mean from winconf-*.cmake

Reply:
>you mean from winconf-*.cmake

yes.

Reply:
done.